src/adaptation/ecap/Host.cc

```diff
@@ -56,7 +56,6 @@ Adaptation::Ecap::Host::Host()
     libecap::protocolHttps.assignHostId(AnyP::PROTO_HTTPS);
     libecap::protocolFtp.assignHostId(AnyP::PROTO_FTP);
     libecap::protocolWais.assignHostId(AnyP::PROTO_WAIS);
-    libecap::protocolUrn.assignHostId(AnyP::PROTO_URN);
```

Comment:
Thus rendering eCAP unable to meet the release notes claimed capability of performing Trivial-HTTP Resolver gateway.
("AnyP::PROTO_UNKNOWN" are passed as static string "unknown", not as the received scheme image)

Reply:
Host application IDs being configured by this code are an optimization that speeds up string comparison for common cases. eCAP code should function correctly without that optimization. If it does not, it is an out-of-scope bug (in eCAP adapter or host application code).